google/auth/jwt_async.py

```diff
@@ -0,0 +1,846 @@
+# Copyright 2016 Google LLC
```

Comment:
Again, this is a huge file-- what changes did you make vs. the sync version that necessitated making a copy?

Reply:
The difference is in the jwt_async file we inherit the service account async credentials - this should be resolved with the refactoring. The reason we still define the async version of the classes when it only uses methods from the sync classes is because when we define an async credentials we keep a separate thread of async credentials.